sphinx/builders/gettext.py

```diff
@@ -53,7 +53,11 @@ def add(self, msg: str, origin: Union[Element, "MsgOrigin"]) -> None:
         if msg not in self.metadata:  # faster lookup in hash
             self.messages.append(msg)
             self.metadata[msg] = []
-        self.metadata[msg].append((origin.source, origin.line, origin.uid))  # type: ignore
+        line = origin.line
+        if line is None:
+            logger.warning(f"Node {origin!r} has no line number, using '-1'.")
```

Comment:
I'm afraid to add a new warning because it also stops CI builds via `-W` option. And there is no way to avoid the warning because this is a bug (?) of docutils or 3rd party extensions. The only way to avoid the crash is not using `-W` option on building gettext message catalogs.

So -1 for adding a warning (especially in patch release).

Reply:
Would you be ok with `info`?

A